src/hookproxy/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+include(TargetExports)
+
+add_library(proxydll SHARED proxydll.cpp hooker.cpp hooker.h mapview.cpp mapview.h sha1.cpp sha1.h)
+target_link_libraries(proxydll PRIVATE base)
+set_target_properties(proxydll PROPERTIES OUTPUT_NAME avifil32 PDB_NAME proxydll)
+target_exports(proxydll SYMBOLS 
+    AVIStreamWrite=FakeAVIStreamWrite
+    AVIFileInit=FakeAVIFileInit
+    AVIFileOpenA=FakeAVIFileOpenA
+    AVIFileRelease=FakeAVIFileRelease
+    AVIFileExit=FakeAVIFileExit
+    AVIFileCreateStreamA=FakeAVIFileCreateStreamA
+    AVIStreamSetFormat=FakeAVIStreamSetFormat
+    AVIStreamRelease=FakeAVIStreamRelease
```

Comment:
Who will call these exports?

Reply:
game.dat calls them, or at the very least imports them. They are just jumps to the real dll when called.